### DISCUSSION TITLE
mocap and send_world_velocity

### DISCUSSION THREAD

**Discussion by fangjir** [bitcraze/discussions#1132] [Q&A]
if i use command "send_velocity_world_setpoint" to control crazyflie , does this mean i don't have to use cf.extpos.send_extpose(x,y,z) with mocap. anyone can help me this problem?

**Comment by ataffanel** [comment]
You likely need to send both:
 - `send_velocity_world_setpoint`: Allows to send a setpoint to the Crazyflie *controller* in order to control the Crazyflie.
 - `cf.extpos.send_extpose(x,y,z)`: Sends the current Crazyflie position to the *estimator* which allows the Crazyflie to estimate its position.

In order to be able to control the Crazyflie in position, it must somehow knows its position. So both packet do work together to achieve autonomous flight.
